Makefile

```diff
@@ -21,23 +21,40 @@ SIMPLENOTE_CHANGES_STD := `find "$(THIS_DIR)" -newer "$(SIMPLENOTE_JS)" \( -name
 SIMPLENOTE_BRANCH = $(shell git --git-dir .git branch | sed -n -e 's/^\* \(.*\)/\1/p')
 
 
-# Build configurations
+##############
+# SIMPLENOTE #
+##############
+
+# Node environment
 NODE_ENV = production
+
 # Defines if we should compile web app via `build` target
 SKIP_BUILD = false
 
+# Host for dev server
+HOST = 0.0.0.0
+
+# Port for dev server
+PORT = 4000
```

Comment:
same as above for these. what is the suggested way to choose a different port, or does it have to be hard-coded?